debugtools/DDR_VM/generate.properties

```diff
@@ -31,7 +31,7 @@ J9Class=true,false
 # This is required to ensure that where a change in a pointer is detected such that a new vX (e.g. v1)
 # version of the field is required, this is always done in the same order.
 
-#TODO: lpgnuyen Don't forget to remove combo specs from here after we delete them
+#TODO: lpnguyen Don't forget to remove combo specs from here after we delete them
```

Comment:
This is somebody's name, and while I can't say if it is correct or not, I don't think it should be changed.

Reply:
I changed it because I'm guessing it's the same as this: https://github.com/eclipse/openj9/search?q=lpnguyen&unscoped_q=lpnguyen